texttospeech/cloud-client/synthesize_file_test.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import synthesize_file
+
+TEXT_FILE = 'resources/hello.txt'
+SSML_FILE = 'resources/hello.ssml'
+
+
+def test_synthesize_text_file(capsys):
+    synthesize_file.synthesize_text_file(text_file=TEXT_FILE)
+    out, err = capsys.readouterr()
+
+    assert 'Audio content written to file' in out
```

Comment:
if possible assert that the output file has size greater than zero.  (doing this in only one of the tests should be enough)

Reply:
done

Reply:
looks good.  just need a new line between import os and import synthesize_file/text.

Reply:
Question: is there a flake8 config file I should grab, that would catch that?

Reply:
Use nox (`pip install nox-automation`; then `nox -l` with grep to find the lint session for your particular subproject, then `nox -s "lint(your-subproject-here)") to use the repo's standard flake8 config.